automatic/wps-office-free/wps-office-free.nuspec

```diff
@@ -34,6 +34,10 @@ Quickly access Cover Pages, Content Pages, and your Header and Footer, or Split
 Avoid the hassle caused by switching between multiple document windows.
 The tab bar that appears above the editable area allows you to switch between documents by Ctrl + Tab.
 This makes your document editing as easy and natural as web browsing.
+* **MultiLanguage:**
```

Comment:
Description changes should be made to the `README.md` file, as the description in the nuspec file is automatically updated from that file.

Please update the readme we these changes instead.